app/components/admin/participants/validation_tasks.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Admin
+  module Participants
+    class ValidationTasks < BaseComponent
+      def initialize(profile:)
+        @profile = profile
+      end
+
+      def render?
+        profile.npq?
+      end
```

Comment:
Can we move this check to the parent component?

Reply:
Ah, you mean the view? I think this is a perfect example when `render?` makes sense - this component is to be rendered for all the types n the future, but currently it only makes sense for `npq` profiles. When the feature is fully complete that method will be gone and `render?` method will be completely removed - which is nice that we have a single place to be removed...

Reply:
Alright, sure.